### PR TITLE
web_editor traceback on saving the product

### PR DESCRIPTION
**Pull request by msh-odoo**
Task:https://www.odoo.com/web#id=2063293&action=333&active_id=1251&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.c134d9b0d8ea08eb773491d23bb96507

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/models/ir_qweb.py [hunk 1]
```diff
@@ -302,7 +302,9 @@ class Image(models.AbstractModel):
 
     @api.model
     def from_html(self, model, field, element):
-        url = element.find('img').get('src')
+        if element.find('.//img') is None:
```
Comment (rdeodoo) [inline addons/web_editor/models/ir_qweb.py]: It is mandatory to change the xpath selector?
Eg why `find('img')` became `find('.//img')`?

Comment (msh-odoo) [reply addons/web_editor/models/ir_qweb.py]: It is due to copy paste from master branch I created previously ;)

**Comment by rdeodoo** [comment]
@JKE-be Can you r+ this one?

**Comment by JKE-be** [comment]
@robodoo r+

**Comment by JKE-be** [comment]
@rdeodoo  and the commit msg :-1:

**Comment by rdeodoo** [comment]
Hm.. As it was a duplicate of https://github.com/odoo/odoo/pull/40467 created afterward I forgot to pay attention to the commit message.

**Pull request closed without merging**